fill/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Load a lightweight Linux image
+FROM ubuntu:22.04 AS run
+
+# Copy the build output from the GitHub Workflow to Alpine
+COPY ./fill/fill_station /bin/fill_station
+
+RUN ["chmod", "+x", "/bin/fill_station/fill_station"]
+
+# Expose the gRPC Server Port
+EXPOSE 50051
+
+# Run the binary
+ENTRYPOINT [ "/bin/fill_station/fill_station" ]
```

Comment:
Have you tested that this is the path?

Reply:
Yes, this is the path. The image works and correctly communicates with the ground gRPC binary